cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html

```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: May 3, 2016</p>
+    <p>Org chart last updated: August 3, 2016</p>
```

Comment:
I double checked with the stakeholder -- this date should be July 26, 2016. @ajbush will make that change in the png and upload it here for you.